include/crm/common/scores_compat.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2024 the Pacemaker project contributors
```

Comment:
may want older start date since `char2score()` is older, if it matters

Reply:
meh, not worth it